### PR TITLE
added a clear section for supported variables

### PR DESCRIPTION
**Pull request by MariePMcGarry**
### Variables

The enrichment server provides allows for these variables:

${IP}, ${MAC}, ${USERNAME}, and ${HOSTNAME} and custom entities

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cse/integrations/insight-enrichment-server.md [hunk 1]
```diff
@@ -133,6 +133,13 @@ Run the installer and follow the instructions.
 
 ## Configuration
 
+### Variables
+
+The enrichment server provides allows for these variables:
+
+${IP}, ${MAC}, ${USERNAME}, and ${HOSTNAME} and custom entities
```
Comment (MariePMcGarry) [inline docs/cse/integrations/insight-enrichment-server.md:140]:
```suggestion
${IP}, ${MAC}, ${USERNAME}, and ${HOSTNAME}, and for custom Entities, ${ENTITY}.
```

File: docs/cse/integrations/insight-enrichment-server.md [hunk 2]
```diff
@@ -133,6 +133,13 @@ Run the installer and follow the instructions.
 
 ## Configuration
 
+### Variables
+
+The enrichment server provides allows for these variables:
```
Comment (MariePMcGarry) [inline docs/cse/integrations/insight-enrichment-server.md:138]:
```suggestion
The Enrichment Server supports these variables:
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Hi @g-wattz , can you please apply Marie's edits on your end and resubmit?

**Review by g-wattz — APPROVED** [review]

**Comment by g-wattz** [comment]
@kimsauce I have approved

**Comment by kimsauce** [comment]
> @kimsauce I have approved

@g-wattz Marie's comments above are still showing as unresolved. You might have to add locally in your fork and re-submit.

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
@g-wattz thank you! Last step - you just have to sync with main branch to get the latest updates, then you can merge.

**Pull request merged**